middlewares/nostrAuth.ts

```diff
@@ -51,7 +51,7 @@ async function validateEvent(
     throw new Error("Invalid nostr event, method tag invalid");
   }
 
-  if (Boolean(body)) {
+  if (Object.keys(body).length) {
```

Comment:
i think it would be better to do the following in case the body is `undefined` which is what i expected it to be.
```suggestion
  if (Boolean(body) && Object.keys(body).length > 0) {
```